client/src/components/DiscoverCarousel/DiscoverCarousel.js

```diff
@@ -74,12 +91,16 @@ const DiscoverCarousel = () => {
         </div>
       </div>
       <div className="nav">
+        {!prevHidden && (
         <button className="prev">
           <ChevronLeftIcon onClick={prevPage} />
```

Comment:
We'll want to move these onClicks one element above, to the button, so it has a bigger click/touch target.